test_requirements.txt

```diff
@@ -1,4 +1,5 @@
 # git+https://github.com/aymgal/coolest@main#egg=coolest
 git+https://github.com/LSSTDESC/OpSimSummaryV2
 coolest==0.1.8
-galsim
\ No newline at end of file
+galsim
+microlensing
```

Comment:
the magnification maps below are quite large (effectively larger than the entire package). Is it possible to have lower-resolution or much smaller maps for the tests?